src/user.ts

```diff
@@ -146,6 +146,36 @@ export class UserApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /**
+   * Function to take the attributes of a new User account, create that
+   * in Marqeta, and return the account information. If no attributes are
+   * provided, a new User account will still be created and its
+   * token Id returned.
+   */
+  async create(user: Partial<User>): Promise<{
+    success: boolean,
+    body?: User,
+    error?: MarqetaError,
+  }> {
+    const searchOptions = snakecaseKeys(user)
```

Comment:
`searchOptions`? This really what you want to call this? Do you really even need it?